content/en/tracing/profiler/getting_started.md

```diff
@@ -76,70 +76,103 @@ The Datadog Profiler requires [JDK Flight Recorder][1]. The Datadog Profiler lib
+**Requirements**
 
-1. If you are already using Datadog, upgrade your agent to version [7.20.2][1]+ or [6.20.2][1]+.
+The Datadog Profiler requires Python 2.7+ and Agent version [7.20.2][1]+ or
+[6.20.2][1]+.
 
-2. Install `ddtrace` which contains both tracing and profiler:
+The following profiling features are available depending on your Python version:
 
-    ```shell
-    pip install ddtrace
-    ```
+|      Feature         | Supported Python versions          |
+|----------------------|------------------------------------|
+| Wall time profiling  | Python >= 2.7                      |
+| CPU time profiling   | Python >= 2.7 on POSIX platforms   |
+| Exception profiling  | Python >= 3.7 on POSIX platforms   |
+| Lock profiling       | Python >= 2.7                      |
+| Memory profiling     | Python >= 3.5                      |
 
-     **Note**: Profiler is available in the `ddtrace` library for versions 0.36+.
+**Installation**
 
-3. To automatically profile your code, set `DD_PROFILING_ENABLED` environment variable to `true` when you use `ddtrace-run`:
+Install `ddtrace`, which provides both tracing and profiling functionalities:
+
+```shell
+pip install ddtrace
+```
+
+**Note**: Profiling requires the `ddtrace` library version 0.40+.
+
+If you're using a platform where `ddtrace` binary distribution is not
+available, you will need to install a development environment.
+
+For example, on Alpine Linux, this can be done with:
+```shell
+apk install gcc musl-dev linux-headers
+```
+
+**Usage**
+
+To automatically profile your code, set the `DD_PROFILING_ENABLED` environment
+variable to `true` when you use `ddtrace-run`:
 
-    ```
     DD_PROFILING_ENABLED=true ddtrace-run python app.py
-    ```
-    **Note:** `DD_PROFILING_ENABLED` is only supported in `dd-trace` version 0.40+. Use the alternate method if you are running an older version of `dd-trace`.
 
-    **Alternate method**
+It is strongly recommended that you add tags like `service` or `version`, as they
+provide the ability to slice and dice your profiles across these dimensions,
+enhancing your overall product experience. See [Configuration][#configuration] below.
 
-    If you prefer to instrument the profiler through code, import `ddtrace.profile.auto`. After import, the Profiler starts:
+After a couple of minutes, visualize your profiles on the [Datadog APM > Profiler page][2].
 
-    ```python
-    import ddtrace.profiling.auto
-    ```
+If you want to manually control the lifecycle of the profiler, use the
+`ddtrace.profiling.profiler.Profiler` object:
 
-4. After a minute or two, visualize your profiles on the [Datadog APM > Profiler page][2].
+```python
+from ddtrace.profiling import Profiler
 
-- It is strongly recommended to add tags like `service` or `version` as it provides the ability to slice and dice your profiles across these dimensions, enhancing your overall product experience. Use environment variables to set the parameters:
+prof = Profiler()
```

Comment:
And we could do the equivalent here, with comments showing what env var is used if not specified, like
```
prof = Profiler(
    env="prod",  # if not specified, uses env var DD_ENV
...
```